src/library/scala/Array.scala

```diff
@@ -55,6 +55,12 @@ object Array {
    */
   def newBuilder[T](implicit t: ClassTag[T]): ArrayBuilder[T] = ArrayBuilder.make[T](t)
 
+  /** Creates an array from a given iterable
+   *
+   * @param it the source iterable
+   * @return an array containing all the elements from it
```

Comment:
Don't hate me, but I understood "it" as a pronoun here. They often put an identifier in backticks to highlight that it is an ident.